nautobot/extras/signals.py

```diff
@@ -298,3 +299,18 @@ def refresh_job_models(sender, *, apps, **kwargs):
             )
             job_model.installed = False
             job_model.save()
+
+
+#
+# Authentication
+#
+
+
+@receiver(user_logged_in)
+def user_logged_in_signal(sender, request, user, **kwargs):
+    logger.info(f"User {user} successfully authenticated")
+
+
+@receiver(user_logged_out)
+def user_logged_out_signal(sender, request, user, **kwargs):
+    logger.info(f"User {user} has logged out")
```

Comment:
Let's put these in `nautobot.core.signals` instead of `nautobot.extras.signals` since this is core functionality rather than an extension point.